bookmarklet.js

```diff
@@ -2677,11 +2677,11 @@ SherdBookmarklet = {
                 asset.sources.poster;
             var newAsset;
             if (img) {
-                newAsset = self.elt(null,'img','sherd-image',{src:img,style:'max-width:215px;max-height:150px',height:null});
```

Comment:
I've updated this commit. The `max-width` wasn't the problem here.. it was a `width: 100% !important` in the CSS which I had addressed in the extension.